README.md

```diff
@@ -1,9 +1,22 @@
 # Bot Sample
 
-A sample bot using the [Microsoft Bot Framework](https://dev.botframework.com) that demonstrates one way to automate bot testing.
+A clear and simple base case bot using the [Microsoft Bot Framework](https://dev.botframework.com) that demonstrates:
+
+* One way to automate bot testing 
+* How to make a bot deployable to Azure Functions, AWS Lambda (via serverless) or restify
+* LUIS.ai integration
+
+This can be useful to those looking to get started quickly with either or all of the three points above. 
+
+To learn how to setup continuous deployment of this bot see: [Continuous deployment for Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-continuous-deployment)
+
+To learn how to setup the necessary environment variables, see: [How to configure Azure Function app settings](https://docs.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings).  
+
+To get the dependent botbuilder package, run npm install within site\wwwroot\[FUNCTION_DIR]
```

Comment:
Hi @vjrantal ! Thanks for taking a look. I tried deploying via Visual Studio Team Services and `npm install` did not run automatically. I might check with the Visual Studio Team Services team but thought I should make a note here and now so that I can start pointing people to this source. I can confirm that deployment via GitHub does indeed resolve those dependencies.

Anyway, good point there. It does make sense for people to not worry about it if there is no need to (i.e. if deployment via GitHub already resolves dependencies). Thanks again!